include/seastar/core/metrics.hh

```diff
@@ -435,8 +435,8 @@ extern label shard_label;
  */
 template<typename T>
 impl::metric_definition_impl make_gauge(metric_name_type name,
-        T&& val, description d=description(), std::vector<label_instance> labels = {}) {
-    return {name, {impl::data_type::GAUGE, "gauge"}, make_function(std::forward<T>(val), impl::data_type::GAUGE), d, labels};
+        T&& val, description d=description(), std::vector<label_instance> labels = {}, std::vector<label> aggregate_labels = {}) {
+    return {name, {impl::data_type::GAUGE, "gauge"}, make_function(std::forward<T>(val), impl::data_type::GAUGE), d, std::move(labels), std::move(aggregate_labels)};
```

Comment:
Looks like this commit isn't needed, it's supposed to be used like:
```c++
make_gauge(...).aggregate(aggregate_labels)
```

Reply:
Makes sense now. I was wondering what the point of that `aggregate` member function was.